src/Schema/Table.php

```diff
@@ -835,34 +779,243 @@ public function __clone()
         }
     }
 
+    public function setComment(?string $comment) : self
+    {
+        // For keeping backward compatibility with MySQL in previous releases, table comments are stored as options.
+        $this->addOption('comment', $comment);
+
+        return $this;
+    }
+
+    public function getComment() : ?string
+    {
+        return $this->_options['comment'] ?? null;
+    }
+
     /**
-     * Normalizes a given identifier.
+     * @return int
+     */
+    protected function _getMaxIdentifierLength()
```

Comment:
Why does the method declaration need to be moved?

Reply:
This method was being moved to follow the logical class declaration order:

* public constants
* protected constants
* private constants
* public static attributes
* protected static attributes
* private static attributes
* public attributes
* protected attributes
* private attributes
* constructor
* static public methods
* static protected methods
* static private methods
* public methods
* protected methods
* private methods
* magic methods

Reply:
For a better wording, the rule to thumb is: Non-public methods should always be at the end of the class declaration.

Reply:
Let's do it in a separate PR. Ideally, it should be done consistently and enforced by the standard.

In this PR, it unnecessarily complicates the review since it introduces new methods and shuffles around the existing ones.

Reply:
@carusogabriel is there something like above in the standard or it could be implemented there?

Reply:
@morozov is that a requirement to have this merged? I'm trying to make ORM `master` point to `3.0.x` instead of `missed-commits`. I did update some methods (don't think this one in particular) because phpstan was not happy.

Reply:
@morozov Yeap, there's: https://github.com/slevomat/coding-standard#slevomatcodingstandardclassesclassstructure-

I don't think that'll make to the next major version of `doctrine/coding-standard`, as that release is already big enough. I'll propose to v9.

Reply:
> […] is that a requirement to have this merged?

I believe it is. Otherwise, I cannot wrap my head around the scope of the PR.

Reply:
I've moved back all methods to their original (not logical) location.